### PR TITLE
Bug fix db integration fix server persistcnce

### PR DESCRIPTION
**Pull request by suryarawat**
- Integrated MongoDB for server persistence

### CODE REVIEW

File: digital-deck-server/models/dbConnection.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require('dotenv').config({path:'./../digital-deck-server/ENV.env'});
```
Comment (mdangnguyen) [inline digital-deck-server/models/dbConnection.js:1]: Does this require anything extra locally? Or should it work automatically?

File: digital-deck-server/services/utilities.js [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
  * @return [array of {json} ] The array of json containing information of all sessions
  */
-function getConcSessions()
+async function getConcSessions()
 {
-    return concSessions;
+    var sessions = await sessionData.find();
+    console.log(sessions);
+    sessions.forEach((session) => {
+        session = cleanSession(session);
```
Comment (mdangnguyen) [inline digital-deck-server/services/utilities.js:24]: Why do we have to do this?

File: digital-deck-server/models/dbConnection.js (hunk 1 above)
Comment (suryarawat) [reply digital-deck-server/models/dbConnection.js:1]: requires a .env file. Which I can share with you. This is for security purposes.

File: digital-deck-server/services/utilities.js (hunk 2 above)
Comment (suryarawat) [reply digital-deck-server/services/utilities.js:24]: When we store session to database it creates it's own _id field which I am removing here before sending it to user

Comment (mdangnguyen) [reply digital-deck-server/services/utilities.js:24]: Ohh I see. So the client should get the correct sessionId in the DB yeah?

Comment (suryarawat) [reply digital-deck-server/services/utilities.js:24]: That and it was unnecessary info for client side. We don't need to remove it but this just keeps the session object identical as before to ensure nothing breaks on UI

**Review by mdangnguyen — APPROVED** [review]

**Review by dhillonJas — APPROVED** [review]

**Pull request merged**